exasol/analytics/schema/column_types.py

```diff
@@ -0,0 +1,119 @@
+import re
+from dataclasses import dataclass
+from enum import (
+    Enum,
+    auto,
+)
+from typing import (
+    Any,
+    Callable,
+    Optional,
+)
+
+
+class CharSet(Enum):
+    UTF8 = auto()
+    ASCII = auto()
+
+    @classmethod
+    def from_string(cls, name: str) -> "CharSet":
+        for c in cls:
+            if c.name == name:
+                return c
+        raise ValueError(f"Couldn't find CharSet with name '{name}'")
+
+
+class PyexasolOption(Enum):
+    CHARACTER_SET = "characterSet"
+    PRECISION = "precision"
+    SCALE = "scale"
+    SIZE = "size"
+    SRID = "srid"
+    UNIT = "unit"
+    WITH_LOCAL_TIME_ZONE = "withLocalTimeZone"
+
+
+@dataclass(frozen=True)
+class PyexasolMapping:
```

Comment:
A docstring would help.

Reply:
Why is it a dataclass?

Reply:
See next push providing a docstring.

Reply:
> Why is it a dataclass?

Maybe I don't understand the question? 

It's a dataclass to hold the attributes as defined. 
Or to put it differently: If you think it should _not_  be a dataclass, then why?

Reply:
This comment helps. Thanks. 
See next push not using a dataclass anymore.